frontend/views/containers/proposals/ProposalItem.vue

```diff
@@ -309,7 +309,7 @@ export default ({
         if (
```

Comment:
the changes in this file is not related to the issue itself but was a fix for a test-failure (in `group-proposals.spec.js`) in `master` branch FYI.